docs/reference/scalars.mdx

```diff
@@ -57,6 +53,10 @@ A unique 32 byte hash identifier for a transaction.
 
 The location of the transaction in the block. It can be used by UTXOs as a reference to the transaction or by the transaction itself to make it unique.
 
+## `U8`
+
+Unsigned 8 bit number.
```

Comment:
Ignore if this is too nitpicky, but I often see this data type referred to as an "integer" not a "number" as it is specifically whole numbers, and as far as I understood, number is more a broad term that can encompass whole numbers and fractions, etc.